server/sonar-server/src/main/java/org/sonar/server/plugins/ServerExtensionInstaller.java

```diff
@@ -59,7 +59,8 @@ public void installExtensions(ComponentContainer container) {
             container.declareExtension(pluginInfo, extension);
           }
         }
-      } catch (Exception e) {
+      } catch (Throwable e) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Catch Exception instead of Throwable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1181)